src/test/rgw/test_rgw_lua.cc

```diff
@@ -280,6 +280,66 @@ TEST(TestRGWLua, URI)
   ASSERT_EQ(rc, 0);
 }
 
+TEST(TestRGWLua, RGWDebugLog2)
+{
+  const std::string script = R"(
+    RGWDebugLog2("Testing {} {}", "hello", 1234567890)
+  )";
+
+  DEFINE_REQ_STATE;
+
+  const auto rc = lua::request::execute(nullptr, nullptr, &s, nullptr, script);
+  ASSERT_EQ(rc, 0);
+}
+
+TEST(TestRGWLua, RGWDebugLog2TooManyArguments)
+{
+  const std::string script = R"(
+    RGWDebugLog2("Testing {}", "hello", 1234567890)
+  )";
+
+  DEFINE_REQ_STATE;
+
+  const auto rc = lua::request::execute(nullptr, nullptr, &s, nullptr, script);
+  ASSERT_EQ(rc, 0);
+}
+
+TEST(TestRGWLua, RGWDebugLog2TooFewArguments)
+{
+  const std::string script = R"(
+    RGWDebugLog2("Testing {} {}", 1234567890)
```

Comment:
should this test pass or fail?
please document the behavior in case of mismatch in number of arguments